pom.xml

```diff
@@ -275,23 +276,29 @@
             <artifactId>org-netbeans-core-multiview</artifactId>
             <version>${netbeans.api.version}</version>
         </dependency>
-        <dependency>
+        <!--        <dependency>
```

Comment:
Copied the file `Yenta.java` to this project. I couldn't find it in a repository.